R/minc_FDR.R

```diff
@@ -58,10 +58,11 @@ mincFDR <- function(buffer, ...) {
 }
 #' @describeIn mincFDR mincSingleDim
 #' @export
-mincFDR.mincSingleDim <- function(buffer, df, mask = NULL, method = "qvalue", ...) {
+mincFDR.mincSingleDim <- function(buffer, df, mask = NULL, method = "fdr", ...) {
   if (is.null(df)) {
     stop("Error: need to specify the degrees of freedom")
   }
+  warning('default method has changed, specify method = "qvalue" to retain old implementation')
```

Comment:
```suggestion
  message('default method has changed, specify method = "qvalue" to retain old implementation')
```

Reply:
Okay, but why not warning? Too strong?

Reply:
Yeah I think so. I'm of the view most warnings should be errors, CRAN for example will fail you if your test suite has any warnings. Also warnings can be promoted to errors with `options(warn = 2)`, a deprecation message should never terminate your program imo.

Reply:
Cool, makes sense.